### DISCUSSION TITLE
Options

### DISCUSSION THREAD

**Discussion by wasabii** [quartznet/quartznet#920] [General] [answered]
As I watch the introduction of `Quartz.Extensions.Hosting` and such, I've noticed a few things wrong with it.

First, it has an Options-style class, but it doesn't use the IOptions injector. Which means services.Configure<It> won't work right.

Second, it tries to go after IConfiguration for "Quartz" on it's own. It shouldn't do that. Should use the Options framework to do configuration, instead of hitting IConfiguration with a hard coded path.

**Comment by wasabii** [comment]
Just thought I'd leave my suggestions. They don't rise to the level of bugs yet. More like bad patterns.

**Comment by lahma** [comment, marked as answer]
There's now improved Options story landing to 3.2 via https://github.com/quartznet/quartznet/pull/978 .
